rsconf/package_data/mongod/mongod.conf.jinja

```diff
@@ -0,0 +1,19 @@
+# for documentation of all options, see:
+#   http://docs.mongodb.org/manual/reference/configuration-options/
+net:
+  port: {{ mongod.port }}
+  bindIp: {{ mongod.ip }}
+processManagement:
+  fork: true
+  pidFilePath: /var/run/mongodb/mongod.pid
```

Comment:
I think this should be /run/mongod/mongod.pid. I have a feeling we need to create /run/mongod, since mongod runs as mongod. Maybe it is created in the rpm? We should use consistent naming (everything mongod) even they don't. /var/run is outdated.

Reply:
6c04a19

/var/run/... is created by the rpm. Just setting pidFilePath to /run/... doesn't work because pidFile in the systemd unit file from the rpm is hard coded to /var/run/... . I think I've resolved all of it by just using a custom unit.

Reply:
I think that's better.